plugins/hydra_ray_launcher/example/conf/config.yaml

```diff
@@ -0,0 +1,6 @@
+defaults:
+  - hydra/launcher: ray_local
+  - ray_mode: local
```

Comment:
Why do have two locals here?

Reply:
Is the problem that the aws overrides in the example are incompatible ray_local?

Reply:
Yea, the ray_mode is for the example app. And yes, the reason for 2 modes is because the config files are not compatible between ray_local and ray_aws